app/src/main/res/values/strings.xml

```diff
@@ -19,4 +19,9 @@
 
     <string name="facebook_app_id">252967458413543</string>
 
+    <string name="yelp_consumer_key">5A05-ep8SUMCsR5UJP-kTQ</string>
```

Comment:
We should probably not be putting up api keys in github 😁 remove from github, generate new keys, and create a secondary strings file that you add to .gitignore - shit wait I just realized we haven't done this to the firebase config files either ugh